src/middlewared/middlewared/plugins/notifier.py

```diff
@@ -230,6 +231,14 @@ def ds_idmap_type_code_to_string(self, code):
             raise ValueError('Unknown idmap code: {0}'.format(code))
         return mapping[code]
 
+    async def ds_clearcache(self):
+        """Temporary call to rebuild DS cache"""
+        await Popen(
```

Comment:
Shouldn't you wait for this process to terminate to avoid keeping zombies?

Reply:
@themylogin I dont think thats necessary since the main thread is the parent and that is not going away

Reply:
@themylogin do you have a concern with this?
I think by the asyncio nature of this the pid will always be waited (In my tests I could not get zombie process anyway), but I can add a task to make sure pid is waited if you think thats really necessary.

Reply:
@william-gr I suppose loop waits for all spawned subprocesses before exiting? If so, I then I think we are done here. I have no detailed experience with subprocesses in asyncio and documentation is very shallow.